client/v3/naming/doc.go

```diff
@@ -39,7 +39,7 @@
 //	func etcdDial(c *clientv3.Client, service string) (*grpc.ClientConn, error) {
 //		etcdResolver, err := resolver.NewBuilder(c);
 //		if err { return nil, err }
-//		return  grpc.Dial("etc://foo", grpc.WithResolvers(etcdResolver))
+//		return  grpc.Dial("etcd://foo", grpc.WithResolvers(etcdResolver))
```

Comment:
Please make it: etcd:///foo

Please see:  `tests/integration/clientv3/naming/resolver_test.go`  as working example.

Reply:
done :-)